doc/source/dev/toolchain.rst

```diff
@@ -252,18 +253,54 @@ the bottleneck for C++ support is therefore the oldest supported GCC version,
 where SciPy has been constrained mainly by the version in the oldest supported
 manylinux versions & images [16]_.
 
-At the end of 2021 (with the final removal of ``manylinux1`` wheels), SciPy
-now has a minimum GCC requirement of GCC 6.3, which has full C++14 support
-[17]_. This corresponds to the lowest present GCC version in relevant manylinux
-versions - somewhat surprisingly, it is not the oldest remaining
-``manylinux2010`` that is the most restrictive (due to the ABI-compatible
-"RHEL Dev Toolset" backports, it has GCC 8.3), but actually ``manylinux_2_24``
-that only comes with GCC 6.3 [18]_.
-
-C++17 _language_ support will require GCC >= 7 (released May 2017). As of the
-end of 2021, support for the entirety of the C++17 standard library has not yet
-been completed across all compilers; similarly, support for C++20 and C++23
-is still under heavy development. [17]_
+At the end of 2021 (with the final removal of ``manylinux1`` wheels), the
+minimal requirement of GCC moved to 6.3, which has full C++14 support [17]_.
+This corresponded to the lowest-present GCC version in relevant manylinux
+versions, though this was still considering the Debian-based "outlier"
+``manylinux_2_24``, which - in contrast to previous manylinux images based on
+RHEL-derivative CentOS that could benefit from the ABI-compatible GCC backports
+in the "RHEL Dev Toolset" - was stuck with GCC 6.3. That image failed to take
+off not least due to those outdated compilers [18]_ and reached its EOL in
+mid-2022 [19]_. For different reasons, ``manylinux2010`` also reached its EOL
+around the same time [20]_.
```

Comment:
Strictly speaking, the past tense is wrong here (as the EOL is August 1st 2022), but I'm aiming for this to be read more after that date than before.

Reply:
That seems fine to me.